backend/apps/medical_catalog/api/views/hospital_item_price_views.py

```
@@ -24,26 +25,28 @@ class HospitalItemPriceViewSet(viewsets.ModelViewSet):
-            instance = HospitalItemPriceService.create(data=request.data, user=request.user)
+            instance = HospitalItemPriceService.hospital_item_price_create(price_data=request.data, user=request.user)
             serializer = self.get_serializer(instance)
             return Response(serializer.data, status=status.HTTP_201_CREATED)
         except ValidationError as e:
-            return Response({'error': 'Validation failed', 'details': e.message_dict}, status=status.HTTP_400_BAD_REQUEST)
+            details = e.message_dict if hasattr(e, 'message_dict') else {'error': str(e)}
+            return Response({'error': 'Validation failed', 'details': details}, status=status.HTTP_400_BAD_REQUEST)
         except Exception as e:
             return Response({'error': 'Invalid data', 'details': str(e)}, status=status.HTTP_400_BAD_REQUEST)
 
     def update(self, request, *args, **kwargs):
```

Ruff (0.14.3)

Unused method argument: `args`

(ARG002)